run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
It doesn't look like we are calling this script from test.sh? I assume you were meaning to do that?

Reply:
I think this is mostly just to spin everything up locally in one script just for local development

Reply:
Yes, this is more for convenience since we need three things running now for BEIMA to work